src/main/java/org/jabref/gui/fieldeditors/contextmenu/CaseChangeMenu.java

```diff
@@ -23,8 +23,7 @@ public CaseChangeMenu(final StringProperty text) {
         Objects.requireNonNull(text);
 
         // create menu items, one for each case changer
-        List<Formatter> caseChangers = new ArrayList<>();
-        caseChangers.addAll(Formatters.getCaseChangers());
+        List<Formatter> caseChangers = new ArrayList<>(Formatters.getCaseChangers());
```

Comment:
I found the previous code easier to understand...

Reply:
IntelliJ reported that as performance issue. I will revert it nevertheless, as I think it's not that crucial...

Reply:
I actually find the intelij variant easy enough to understand. It's obvious that the constructor of ArrayList can take a collection